arch/c4_context.md

```diff
@@ -6,40 +6,70 @@ All production Systems (External and Radix) and Users that are commonly found on
 
 ![c4_context.puml](http://www.plantuml.com/plantuml/proxy?cache=no&fmt=svg&src=https://raw.githubusercontent.com/radixdlt/docs/arch/arch/c4_context.puml)
 
+# Universes
+
+Described in detail in the Radix [Whitepaper v2](https://papers.radixdlt.com/tempo/v2/).
+
+From an architectural point of view, a Universe identifies and overlay network on top of Internet or in a LAN.
+Core/Boot nodes can be configured to be part of a specific universe.
+
+# Atoms and Particles
+
+Described in detail in the Radix [Whitepaper v2](https://papers.radixdlt.com/tempo/v2/).
+
+On an Architectural level an Atom is totally ordered sequence of Transactions (Particles) that must be validated as a unit and committed to the Ledger (DB) atomically. 
```

Comment:
Mhmmm don't think this is quite correct. Something like:
An Atom is a database transaction containing a totally ordering sequence of state transitions (i.e. Particles).